multicast/exceptions.py

```
@@ -231,11 +231,11 @@ class CommandExecutionError(RuntimeError):
 			1
 	"""
 
-	__module__ = """multicast.exceptions"""
+	__module__ = "multicast.exceptions"
 
-	__name__ = """multicast.exceptions.CommandExecutionError"""
+	__name__ = "multicast.exceptions.CommandExecutionError"
 
-	def __init__(self, *args, **kwargs):
+	def __init__(self, *args, **kwargs) -> None:
```

Ruff (0.8.2)

Missing type annotation for `*args`

(ANN002)